.github/workflows/release.yml

```diff
@@ -1,4 +1,4 @@
-name: Cli Release
+name: CLI Release
```

Comment:
If we change this name, I think we need to update the places like [this](https://github.com/twilio/twilio-cli/blob/main/.github/workflows/rpmbuild.yml#L7) where we are using the name of this workflow.

Reply:
Will revert it then for uniformity.